### PR TITLE
Cmdline fixes

### PR DESCRIPTION
**Pull request by akallabeth**

### CODE REVIEW

File: channels/tsmf/client/tsmf_media.c [hunk 1]
```diff
@@ -480,12 +480,12 @@ static BOOL tsmf_sample_playback_video(TSMF_SAMPLE* sample)
...
 		static int frame_id = 0;
 		char buf[100];
-		FILE* fp;
 
 		if ((frame_id % 30) == 0)
 		{
 			sprintf_s(buf, sizeof(buf), "/tmp/FreeRDP_Frame_%d.ppm", frame_id);
-			fp = fopen(buf, "wb");
+			FILE* fp = fopen(buf, "wb");
+            if (fp) {
```
Comment (mfleisz) [inline channels/tsmf/client/tsmf_media.c:488]: Formatting screwed

File: client/common/client_cliprdr_file.c [hunk 2]
```diff
@@ -1537,23 +1537,20 @@ static void cliprdr_local_file_try_close(CliprdrLocalFile* file, UINT res, UINT6
...
 		WLog_Print(file->context->log, WLOG_DEBUG, "closing file %s after read", file->name);
-		fclose(file->fp);
-		file->fp = NULL;
 	}
 	else
 	{
 		// TODO: we need to keep track of open files to avoid running out of file descriptors
 		// TODO: for the time being just close again.
+	}
+	if (file->fp)
```
Comment (mfleisz) [inline client/common/client_cliprdr_file.c:1551]: Either `{}` missing or formatting

File: libfreerdp/codec/test/TestFreeRDPCodecRemoteFX.c [hunk 3]
```diff
@@ -877,8 +877,10 @@ int TestFreeRDPCodecRemoteFX(int argc, char* argv[])
 
 #if 0
 	FILE *f = fopen("/tmp/windows.data", "w");
+    if (f) {
```
Comment (mfleisz) [inline libfreerdp/codec/test/TestFreeRDPCodecRemoteFX.c:880]: Formatting

**Review by mfleisz — COMMENTED** [review]
Overall lgtm - some formatting issues (wonder why the checker didn't point that out)

**Comment by akallabeth** [comment]
@mfleisz ok, strange, should have picked up these changes, ok, done manually.

File: client/common/client_cliprdr_file.c (hunk 2 above)
Comment (akallabeth) [reply client/common/client_cliprdr_file.c:1551]: fixed indent of following line.

**Review by hardening — APPROVED** [review]

**Pull request merged**